src/Avalonia.Themes.Default/Accents/BaseDark.xaml

```diff
@@ -52,5 +52,8 @@
         <sys:Double x:Key="FontSizeSmall">10</sys:Double>
         <sys:Double x:Key="FontSizeNormal">12</sys:Double>
         <sys:Double x:Key="FontSizeLarge">16</sys:Double>
+
+        <sys:Double x:Key="ScrollBarWidth">10</sys:Double>
```

Comment:
Do we need a separate width and height? Is there a case where we'd want horizontal scrollbars to be a different height to the vertical scrollbar width?

Reply:
Will test both cases and see what works best.